test/modules/aws-account-integration.ts

```diff
@@ -21,7 +22,7 @@ describe('AwsAccount Integration Testing', () => {
   // TODO: Restore some mechanism to verify credentials
   /*it('does not create a test DB with fake credentials', (done) => void iasql.add(
     dbAlias,
-    'us-west-2',
+    getRandomRegion(),
```

Comment:
You need to do this at the top of the test one time, or it'll do different regions for the different sets of tests and could cause failures

Reply:
Just for this test, afaict